### ISSUE TITLE
Uncaught TypeError: Cannot read property '0' of null

### ISSUE THREAD

**Issue by tscholak** [willwhitney/hydrogen#44]
steps to reproduce:

1. open a python file from the command line, e.g. `atom foo.py`
2. add `# -*- coding: utf-8 -*-` to the beginning of the file
3. move the cursor to the beginning of the file
4. choose `hydrogen: run` from the command palette or press ⌘-⌥-↩

**Atom Version**: 0.201.0
**System**: Mac OS X 10.10.4
**Thrown From**: [Hydrogen](https://github.com/willwhitney/hydrogen) package, v0.3.10

### Stack Trace

Uncaught TypeError: Cannot read property '0' of null

```
At /Users/tscholak/.atom/packages/Hydrogen/lib/main.coffee:159

TypeError: Cannot read property '0' of null
    at /Users/tscholak/.atom/packages/Hydrogen/lib/main.coffee:186:54
    at /Users/tscholak/.atom/packages/Hydrogen/lib/main.coffee:231:56
    at /Users/tscholak/.atom/packages/Hydrogen/lib/config-manager.coffee:34:18
    at Object.module.exports.portfinder.findManyHelper (/Users/tscholak/.atom/packages/Hydrogen/lib/find-port.coffee:29:16)
    at /Users/tscholak/.atom/packages/Hydrogen/lib/find-port.coffee:36:26
    at Server.<anonymous> (/Users/tscholak/.atom/packages/Hydrogen/lib/find-port.coffee:19:20)
    at Server.g (events.js:257:16)
    at emitNone (events.js:67:13)
    at Server.emit (events.js:163:7)
    at net.js:1443:10
```

### Commands

```
     -0:03.8.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:00.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:00.7.0 hydrogen:run (atom-text-editor.editor)
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
Hydrogen, v0.3.10

# Dev
No dev packages
```

**Comment by rgbkrk** [comment]
Can also open a blank file and run it.

**Comment by willwhitney** [comment]
Just published v0.3.11 to address this. Now if you try to run an empty file it does nothing.
